### PR TITLE
adding HCU transaction for Sept 18th 2023

### PR DESCRIPTION
**Pull request by vishalchangrani**

### CODE REVIEW

File: transactions/set-version-boundary/2023/sep-18/arguments.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "type": "UInt8",
+    "value": "17"
+  },
+  {
+    "type": "Optional",
+    "value": "patch.1"
+  },
+  {
+    "type": "UInt64",
+    "value": "???"
```
Comment (vishalchangrani) [inline transactions/set-version-boundary/2023/sep-18/arguments.json:20]: this is TBD

**Review by joshuahannan — APPROVED** [review]
Seems fine to me, but I'm not very familiar with the `NodeVersionBeacon` smart contract, so we might want to have janez or Gio look at it

**Comment by vishalchangrani** [comment]
> Seems fine to me, but I'm not very familiar with the `NodeVersionBeacon` smart contract, so we might want to have janez or Gio look at it

@janezpodhostnik / @sisyphusSmiling mind reviewing the PR real quick?

File: transactions/set-version-boundary/2023/sep-18/arguments.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "type": "UInt8",
+    "value": "31"
+  },
+  {
+    "type": "UInt8",
+    "value": "17"
+  },
+  {
+    "type": "Optional",
+    "value": "patch.1"
```
Comment (sisyphusSmiling) [inline transactions/set-version-boundary/2023/sep-18/arguments.json:16]: Not entirely sure as I don't work with Cadence JSON much, but I think this should be:
```suggestion
    "value": {
      "type": "String,
      "value": "patch.1"
```
Based on [these docs](https://developers.flow.com/cadence/json-cadence-spec#optional) for non-nil optional values

Comment (vishalchangrani) [reply transactions/set-version-boundary/2023/sep-18/arguments.json]: updated...thanks for flagging it.

**Pull request merged**